docs/content/manual/pre-release/ha/single-replica-node-down.md

```diff
@@ -30,4 +31,4 @@ https://github.com/longhorn/longhorn/issues/2309
 7. Power on node of volume/replica.
 8. Verify the auto salvage finishes for volumes.
 9. Verify volume attached & accessible by pod (i.e test data is available).
- 
\ No newline at end of file
+   - For volume which data locality = strict-local, verify that volume/replica/pod still work on original node.
```

Comment:
If data locality is not strict-local, the volume/replica will run on another node, not the original node?

Reply:
Sorry this part is redundant, I removed that.